src/components/views/TasksManager.tsx

```diff
@@ -137,44 +154,122 @@ const TasksManager = (): JSX.Element => {
+            tasks={tasksArray}
+            addComponent={add}
+            addTask={addTask}
+            updateTask={updateTask}
+            deleteTask={deleteTask}
+            members={members}
+            onHide={handleHide}
+          />
+        </Grid>
+      </Slide>
+    );
+  };
+
+  const renderTasksListMinimized = (
+    title: string,
+    label: string,
+  ): JSX.Element => {
+    // eslint-disable-next-line react/destructuring-assignment
+    const tasksArray = tasks.filter(({ data }) => data.label === label);
+    const isHidden = hiddenLists.get(label) ?? false;
+
+    const handleShow = (): void => {
+      setHiddenLists(hiddenLists.set(label, false));
+    };
+
+    return (
+      <Slide
+        direction="up"
+        in={isHidden}
+        mountOnEnter
+        unmountOnExit
+        style={{ height: 'auto' }}
+      >
+        <TasksListMinimized
           title={title}
-          label={label}
           tasks={tasksArray}
-          addComponent={add}
-          addTask={addTask}
-          updateTask={updateTask}
-          deleteTask={deleteTask}
-          members={members}
+          onShow={handleShow}
         />
-      </Grid>
+      </Slide>
     );
   };
 
   return (
-    <Grid container spacing={4} height="100%">
+    <Grid
+      container
+      spacing={4}
+      maxHeight="100vh"
+      position="relative"
+      zIndex={0}
+    >
       <Grid item sm={12} md={2} height="100%">
         <MembersList members={members} />
+        {renderTasksListMinimized(t('todo'), TASK_LABELS.TODO)}
+        {renderTasksListMinimized(t('in_progress'), TASK_LABELS.IN_PROGRESS)}
+        {renderTasksListMinimized(t('completed'), TASK_LABELS.COMPLETED)}
       </Grid>
       <Grid item sm={12} md={10}>
-        <DndContext onDragEnd={handleDragEnd} sensors={sensors}>
-          <Grid container spacing={2} height="100%">
-            {renderTasksList(t('To Do'), TASK_LABELS.TODO, true)}
-            {renderTasksList(t('In Progress'), TASK_LABELS.IN_PROGRESS)}
-            {renderTasksList(t('Completed'), TASK_LABELS.COMPLETED)}
-          </Grid>
+        <DndContext
+          onDragStart={handleDragStart}
+          onDragEnd={handleDragEnd}
+          sensors={sensors}
+        >
+          <TransitionGroup>
+            <Grid container spacing={2}>
+              {renderTasksList(t('todo'), TASK_LABELS.TODO, true)}
+              {renderTasksList(t('in_progress'), TASK_LABELS.IN_PROGRESS)}
+              {renderTasksList(t('completed'), TASK_LABELS.COMPLETED)}
+            </Grid>
+          </TransitionGroup>
+          <DragOverlay dropAnimation={null}>
+            {activeTask ? (
+              <Task
+                key={0}
+                task={activeTask}
+                updateTask={(): void => {
+                  throw new Error('Function not implemented.');
+                }}
+                deleteTask={(): void => {
+                  throw new Error('Function not implemented.');
+                }}
```

Comment:
Finish implementation ? Or are these not used ?

Reply:
These are not used. But I could just put empty functions.

Reply:
Yes, or better, make the props optional, if the component can work without them.

Reply:
Is it better in your opinion?

Reply:
I did change the component in the end. Thanks for the help.